canvas_modules/harness/cypress/support/index.js

```diff
@@ -28,6 +28,7 @@ import "./canvas/toolbar-cmds";
 import "./canvas/utils-cmds";
 import "./canvas/verification-cmds";
 import "./canvas/operation-cmds";
+import "../integration/canvas/bottom-panel";
```

Comment:
Why is this here? Do we need it? We don't list any of the other 'integration' files here which ought to be a clue that bottom-panel is not needed here.  This seems to be causing the testcase to run for a second time. Are you not seeing that when you run the bottom-panel test in Cypress?